pages/api/report.ts

```diff
@@ -0,0 +1,112 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import dbConnect from '../../middleware/database';
+import { getSession } from 'next-auth/react';
+import Report from '../../models/Report';
+import { Octokit } from 'octokit';
+import hackathon from '../../models/hackathon';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse<any>) {
+	const session = await getSession({ req });
+
```

Comment:
if session userType doesn't exist, should `return res.status(403).send('Forbidden');`